dev/com.ibm.ws.transport.http2_fat/test-applications/H2FATDriver.war/src/http2/test/driver/war/servlets/H2FATDriverServlet.java

```diff
@@ -1654,6 +1709,14 @@ public void testDynamicTableSize(HttpServletRequest request,
         CountDownLatch blockUntilConnectionIsDone = new CountDownLatch(1);
         Http2Client h2Client = getDefaultH2Client(request, response, blockUntilConnectionIsDone);
 
+        // Netty expecting a reset stream because the headers sent back exceed the 128 the number specified by the settings
```

Comment:
Clean this up if not necessary

Reply:
@volosied

Reply:
I'm leaning on keeping this comment as is since the problem is with the Channel Framework. 

However, sending the reset frame could be considered a breaking change if we add it now (even though it's required per the RFC) 

`https://github.com/OpenLiberty/open-liberty/issues/29561` is still open, but we can close it for Netty and open a CHFW issue instead? 

Although, Netty was updated via an overlay match CHFW via https://github.com/OpenLiberty/open-liberty/commit/5199c3f3764f815040a41cf80e1f495b134dfd0e#diff-1bae1b947426d4bb846211a37f4544ccccdbff6a90e5511d47a89df70b61a172R181 

My understanding of this issue, at least.